aten/src/ATen/native/native_functions.yaml

```diff
@@ -1217,6 +1217,7 @@
 
 - func: clamp_max.Tensor(Tensor self, Tensor max) -> Tensor
   variants: function, method
+  structured_delegate: clamp_max.Tensor_out
   dispatch:
     CompositeExplicitAutograd: clamp_max
```

Comment:
you can also get rid of these CompositeExplicitAutograd entries (`clamp_max`/`clamp_max_`/`clamp_min`/`clamp_min_`) since they get codegen'd!

Reply:
Oh nice, so are they authomatically codegened if there's clamp_max in derivatives.yaml?

Reply:
Yep! Structured kernel operators should basically always have a derivative formula already if we want them to work with autograd, since `CompositeImplicitAutograd` ops don't currently work with the structured kernel codegen.

 We generate a `CompositeExplicitAutograd` kernel that allocates an output using `at::empty()`, and then basically just calls into the `TORCH_IMPL_FUNC` out= logic. Like:

```
at::Tensor wrapper_sgn(const at::Tensor & self) {
  structured_sgn_default_backend_functional op;
  op.meta(self);
  at::sgn_outf(self, *op.outputs_[0]);
  return std::move(op.outputs_[0]).take();
}
```